### PR TITLE
add guide for application configuration

### LINKED ISSUE #1428: write doc for creating export config and sharing signing keys

### PR DESCRIPTION
**Pull request by mikehelmick**
Fixes #1428 

## Proposed Changes

* Add configuration docs

**Release Note**

```release-note
Add documentation for e2e application configuration.
```

### CODE REVIEW

File: docs/getting-started/configure_application.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
+---
+layout: default
+---
+<!-- TOC depthFrom:2 depthTo:6 orderedList:false updateOnSave:true withLinks:true -->
+
+- [Before You Begin](#before-you-begin)
```
Comment (whaught) [inline docs/getting-started/configure_application.md:6]: Super nit: your titles are inconsistently capitalized

File: docs/getting-started/configure_application.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+![realm settings](../images/application02.png)
+
+### Configure Realm
+
+These are the recommended settings in tha general realm configuration, you are
+free to deviate from this.
+
+* On the __General__ tab
+    * Select `Enable key-server statistics`
+    * leave everything as as defaults.
```
Comment (whaught) [inline docs/getting-started/configure_application.md:86]:
```suggestion
    * Leave everything as as defaults.
```

**Review by mariliamelo — APPROVED** [review]

File: docs/getting-started/configure_application.md (hunk 1 above)
Comment (mikehelmick) [reply docs/getting-started/configure_application.md:6]: https://github.com/google/exposure-notifications-server/pull/1431

File: docs/getting-started/configure_application.md (hunk 2 above)
Comment (mikehelmick) [reply docs/getting-started/configure_application.md:86]: https://github.com/google/exposure-notifications-server/pull/1431

**Pull request merged**